token/cli/src/main.rs

```diff
@@ -677,15 +692,21 @@ fn main() {
         let json_rpc_url = value_t!(matches, "json_rpc_url", String)
             .unwrap_or_else(|_| cli_config.json_rpc_url.clone());
 
-        let client_keypair = || {
-            read_keypair_file(&cli_config.keypair_path).unwrap_or_else(|err| {
-                eprintln!("Unable to read {}: {}", cli_config.keypair_path, err);
-                exit(1)
-            })
-        };
-
-        let owner = keypair_of(&matches, "owner").unwrap_or_else(client_keypair);
-        let fee_payer = keypair_of(&matches, "fee_payer").unwrap_or_else(client_keypair);
+        let mut wallet_manager = None;
+        let owner = signer_from_path(
+            &matches,
+            &cli_config.keypair_path,
+            "owner",
+            &mut wallet_manager,
+        )
+        .unwrap();
```

Comment:
Does this `unwrap()` panic on an invalid path?  If so please `eprintln!()` the error and `exit(1)` cleanly.  Same with the next one

Reply:
Done!